archive/100-theorems-list/37_solution_of_cubic.lean

```diff
@@ -0,0 +1,201 @@
+/-
+Copyright (c) 2021 Joeff Lee. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: -
```

Comment:
```suggestion
Authors: Jeoff Lee
```
Unless you have a reason you'd rather not be listed here, you did author this file so it seems natural to have your name here.